downloads/swan-lake-release-notes/swan-lake-alpha2/RELEASE_NOTE.md

```diff
@@ -2,41 +2,599 @@
+                             OVERWRITE) returns Error? {
+}
+```
+**Example**
+
+```ballerina
+import ballerina/io;
+
+public function main() returns error? {
+    xml content = xml `<note>
+        <to>Tove</to>
+        <from>Jani</from>
+        <heading>Reminder</heading>
+        <body>Don't forget me this weekend!</body>
+        </note>`;
+    string publicId = "-//W3C//DTD HTML 4.01 Transitional//EN";
+    string systemId = "http://www.w3.org/TR/html4/loose.dtd";
+    var writeResult = io:fileWriteXml("./xmlFileWithDoc.xml", content, doctype = {
+        system: systemId,
+        'public: publicId
+    });
+}
+```
+
+**Output**
+```xml
+<!DOCTYPE note PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<note>
+    <to>Tove</to>
+    <from>Jani</from>
+    <heading>Reminder</heading>
+    <body>Don't forget me this weekend!</body>
+</note>
+```
+##### Email Module Improvements
+
+- Make the `body` field of the `email:Message` record optional. This enables sending an email with only the `htmlBody` field set without a text-typed `body` field.
+
+- Add `mime:Entity` type to the union type of the `attachments` field in the `email:Message` record. That enables attaching a single MIME Entity as an attachment. This `email:Message` record change would appear as follows.
+
+```ballerina
+public type Message record {|
+    // ... other fields
+    mime:Entity|Attachment|(mime:Entity|Attachment)[] attachments?;
+|};
+```
+
+- The above-mentioned changes related to the `body` and `attachments` fields are updated in the `email:Options` record as given below in order to facilitate the same in the `sendEmail` method in the `email:SmtpClient`.
+
+```ballerina
+public type Options record {|
+    // ... other fields
+    string body?;
+    mime:Entity|Attachment|(mime:Entity|Attachment)[] attachments?;
+|};
+```
+
+- Remove the `properties` field representing custom properties from all Email module related configuration records: `email:SmtpConfig`, `email:PopConfig`, `email:ImapConfig`, `email:PopListenerConfig`, and `email:ImapListenerConfig`.
+
+- The `mail.smtp.ssl.checkserveridentity` custom property was passed as an entry in the `properties` to enable/disable the server certificate’s hostname verification. As the `properties` field is removed from the API, from this release onwards, the `verifyHostName` boolean field is introduced to the `secureSocket` record for all the configurations related to the Email module. 
+
+This `email:SecureSocket` record change would appear as follows.
+
+```ballerina
+public type SecureSocket record {|
+    // ... other fields
+    boolean verifyHostname = true;
+|};
+```
+
+##### TCP Module Improvements
+
+Introduced SSL/TLS-based communication to the TCP module using the `secureSocket` record configuration in both the client and the listener.
+
+###### TCP Client with `secureSocket`
```

Comment:
I think this needs to be reworded. `secureSocket` is the param name right? Wouldn't "Secure TCP Client" or "Support for Securing TCP Client" be more appropriate? @shafreenAnfar @chamil321

Reply:
@Maninda

Reply:
@MohamedSabthar

Reply:
Need to remove the `tcp module improvement` section

Reply:
Removed.